target/linux/ramips/base-files/etc/board.d/02_network

```diff
@@ -201,6 +201,11 @@ ramips_setup_interfaces()
 	wl-330n)
 		ucidef_set_interface_lan "eth0.1"
 		;;
+	mr200)
+		ucidef_add_switch "switch0" \
+			"0:lan" "1:lan" "2:lan" "3:lan" "6t@eth0"
+		ucidef_set_interface_wan "usb0"
+		;;
```

Comment:
We also likely need to manually specify usb0 ip address to 192.168.225.100/24 instead of relying on dhcp. DMZ is set on that address and the modem's dhcp server does not seem to be giving that address to the router as it's first choice. And of course if we do so, we also have to manually specify the gateway and dns.
what are your thoughts on this?